docs/eventing/channels/channels.yaml

```diff
@@ -42,3 +18,9 @@ channels:
     support: None
     description: >
       Channels are backed by [NATS Streaming](https://github.com/nats-io/nats-streaming-server#configuring).
+  - name: CRD - GCP PubSub
```

Comment:
I'd prefer to remove the CRD prefix from all these since they are the only ones left.

Reply:
Good idea 👍